packages/blocks/src/blocks/index.ts

```diff
@@ -63,4 +62,37 @@ const blocks: BlockDefinitions = {
 	default: Default,
 };
 
+export type BlockDefinitions = typeof blocks;
```

Comment:
What does this mean for adding custom blocks that we haven't bundled by default?

Reply:
Updated the type for `BlockDefinitions`